org.gephi.Gephi.yml

```diff
@@ -0,0 +1,41 @@
+app-id: org.gephi.Gephi
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: gephi
+
+finish-args:
+  - --device=dri
+  - --filesystem=home
+  - --share=network
+  - --share=ipc
+  - --socket=x11
+
+modules:
+  - name: gephi
+    buildsystem: simple
+    build-commands:
+      - tar -xf gephi.tar.gz
+      - rm -rf gephi.tar.gz
+      - mv gephi-*/{bin,etc,extra,gephi,jre-x64,platform} /app
+      - install -D --mode=0644 gephi.png /app/share/icons/hicolor/512x512/apps/org.gephi.Gephi.png
+      - install -D --mode=0644 --target-directory=/app/share/applications/ org.gephi.Gephi.desktop
+      - install -D --mode=0644 --target-directory=/app/share/metainfo/ org.gephi.Gephi.metainfo.xml
+    sources:
+      - type: file
+        url: https://github.com/gephi/gephi/releases/download/v0.9.7/gephi-0.9.7-linux-x64.tar.gz
+        sha256: 082c4015454ac0ac52ec50b4c97ed7db7810007e18db7beb05e65a11ac19a9c5
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/gephi/gephi/releases/latest
+          version-query: .tag_name | gsub("v";"")
+          url-query: '"https://github.com/gephi/gephi/releases/download/v" + $version + "/gephi-" + $version + "-linux-x64.tar.gz"'
+        dest-filename: gephi.tar.gz
+        only-arches:
+          - x86_64
+      - type: file
+        path: org.gephi.Gephi.desktop
+      - type: file
+        path: org.gephi.Gephi.metainfo.xml
+      - type: file
+        path: gephi.png
```

Comment:
Ideally, you should maintain these files at https://github.com/gephi/gephi/ and make them part of the tarball used for building.

Reply:
Understood. We're about to release a 0.10.0 version actually so let me do that now and change this PR so we don't have to do another PR in a week or so.

Reply:
No worries, it's not a blocker. Just something to address in the future.